packages/labs/compiler/src/lib/template-transform.ts

```diff
@@ -290,6 +290,16 @@ class CompiledTemplatePass {
             shouldCompile = false;
             return false;
           }
+          if (
+            /^(?:textarea|template)$/i!.test(node.tagName) &&
```

Comment:
This regex works, but it's somewhat tricky! tagName is always all caps, so we don't need to worry about that. A regexp can be faster if it doesn't have to ignore case, so consider writing it with all caps. But since this code isn't _that_ performance sensitive, consider just checking membership in a set.

Breaking that part of the conditional out into a local variable also lets us name it, which is helpful for documentation.

Looking at the HTML spec, for elements with a Text content model, I wonder if we should also check for `<title>`, `<style>`, `<rp>`, and an `<option>` with a label attribute but not a value attribute

Reply:
Is this something to improve in lit-html as well (regarding the other elements)?

I can refactor compiler.